tests/gordo_components/dataset/test_sensor_tag.py

```diff
@@ -7,28 +7,30 @@
 from gordo_components.dataset.sensor_tag import SensorTag
 
 
-TAG_NAME1 = "MyBeautifulTag1"
-TAG_NAME2 = "MyBeautifulTag2"
+NON_RESOLVABLE_TAG_NAME1 = "MyBeautifulTag1"
+NON_RESOLVABLE_TAG_NAME2 = "MyBeautifulTag2"
 asset_nonsense = "ImaginaryAsset"
 
 
 @pytest.mark.parametrize(
-    "good_input_tags,asset,expected_output_tags",
+    "good_input_tags,asset,default_asset,expected_output_tags",
     [
         (
             [
-                {"name": TAG_NAME1, "asset": asset_nonsense},
-                {"name": TAG_NAME2, "asset": asset_nonsense},
+                {"name": NON_RESOLVABLE_TAG_NAME1, "asset": asset_nonsense},
+                {"name": NON_RESOLVABLE_TAG_NAME2, "asset": asset_nonsense},
             ],
-            "ThisAssetCodeWillBeIgnored",
+            "ThisAssetCodeWillBeIgnoredSinceAssetMustBeProvidedInDict",
+            "AsWillThisSinceAssetMustBeProvidedInDict",
             [
-                SensorTag(TAG_NAME1, asset_nonsense),
-                SensorTag(TAG_NAME2, asset_nonsense),
+                SensorTag(NON_RESOLVABLE_TAG_NAME1, asset_nonsense),
+                SensorTag(NON_RESOLVABLE_TAG_NAME2, asset_nonsense),
             ],
         ),
         (
             ["TRC-123", "GRA-214", "ASGB-212"],
             "ThisWillBeTheAsset",
+            "AndThisWIllBeIgnoredSinceAllTagsAreResolvable",
```

Comment:
Nit pick of nit picks: `..Will..`

Reply:
damn, you actually read those?

Reply:
I read your code like it was written by the hand of god.